### PR TITLE
Add link to reference implementation (project portal open source project)

### PR DESCRIPTION
**Pull request by Michadelic**
SAP's implementation for the InnerSource portal pattern is now open source. This change adds a link and some context to the pattern body. Enjoy ;-)

### CODE REVIEW

File: patterns/2-structured/innersource-portal.md [hunk 1]
```diff
@@ -34,7 +34,9 @@ Your organization is interested in adopting an InnerSource work style and InnerS
 
 ## Solutions
 
-Create an InnerSource Portal intranet website where InnerSource project owners can easily advertise the availability of their projects.
+Create an InnerSource Portal intranet website where InnerSource project owners can easily advertise the availability of their projects. 
+
+A [reference implementation](http://github.com/sap/project-portal-for-InnerSource) for implementing this pattern is available on GitHub and open for contributions. It Lists all InnerSource projects of a company in an interactive and ease to use way. Projects can self-register using a dedicated GitHub topic and provide additional metadata.
```
Comment (spier) [inline patterns/2-structured/innersource-portal.md]: Fixing two typos and one repetition.

```suggestion
A [reference implementation](http://github.com/sap/project-portal-for-InnerSource) for this pattern is available on GitHub and open for contributions. It lists all InnerSource projects of a company in an interactive and easy to use way. Projects can self-register using a dedicated GitHub topic and provide additional metadata.
```

File: patterns/2-structured/innersource-portal.md [hunk 2]
```diff
@@ -47,7 +49,7 @@ The InnerSource Portal has enabled InnerSource project owners to advertise their
...
-* SAP promotes InnerSource projects in the InnerSource portal - projects can self-register using GitHub topics. The [Repository Activity Score](repository-activity-score.md) defines the default order of the InnerSource projects in the portal. Also see [Michael Graf & Harish B (SAP) at ISC.S11 - The Unexpected Path of Applying InnerSource Patterns](https://www.youtube.com/watch?v=6r9QOw9dcQo&list=PLCH-i0B0otNQZQt_QzGR9Il_kE4C6cQRy&index=6).
+* SAP promotes InnerSource projects in the InnerSource portal - projects can self-register using GitHub topics. The [Repository Activity Score](repository-activity-score.md) defines the default order of the InnerSource projects in the portal. Also see [Michael Graf & Harish B (SAP) at ISC.S11 - The Unexpected Path of Applying InnerSource Patterns](https://www.youtube.com/watch?v=6r9QOw9dcQo&list=PLCH-i0B0otNQZQt_QzGR9Il_kE4C6cQRy&index=6). It's codebase is published as a [reference implementation](http://github.com/sap/project-portal-for-InnerSource) and open for contributions.
```
Comment (spier) [inline patterns/2-structured/innersource-portal.md:52]:
```suggestion
* SAP promotes InnerSource projects in the InnerSource portal - projects can self-register using GitHub topics. The [Repository Activity Score](repository-activity-score.md) defines the default order of the InnerSource projects in the portal. Also see [Michael Graf & Harish B (SAP) at ISC.S11 - The Unexpected Path of Applying InnerSource Patterns](https://www.youtube.com/watch?v=6r9QOw9dcQo&list=PLCH-i0B0otNQZQt_QzGR9Il_kE4C6cQRy&index=6). It's codebase is published as a [reference implementation](http://github.com/sap/project-portal-for-InnerSource) and is open for contributions.
```

**Review by spier — APPROVED** [review]
I added two suggested changes and typo fixes.
Otherwise this looks good to me.

Props for open sourcing this!!!

File: patterns/2-structured/innersource-portal.md (hunk 1 above)
Comment (Michadelic) [reply patterns/2-structured/innersource-portal.md]: awesome, thx!

**Review by lenucksi — APPROVED** [review]
LGTM, thanks for OSS'ing it @Michadelic !

**Pull request merged**